holmes/plugins/toolsets/infrainsights/mongodb_toolset.py

```
@@ -0,0 +1,524 @@
+            # Get collections
+            collections = db.list_collection_names()
+            
+            # Get collection stats
+            result = {
+                "database": database_name,
+                "collections": []
+            }
+            
+            for collection_name in collections:
+                try:
+                    collection = db[collection_name]
```

Ruff (0.12.2)

Local variable `collection` is assigned to but never used

Remove assignment to unused variable `collection`

(F841)